docs/UI/ScrollableMessage/index.md

```diff
@@ -51,24 +51,23 @@ ScrollableMessage with STEAL_FOCUS button for background application
   {
     "messageText" :
     {
-         "fieldName" :  scrollableMessageBody,
-     "fieldText" : "Create a Station
-          Enter an artist, song or composer in the Search box in the top left corner. We'll create a radio station featuring that music and more like it. You can also create a new station from the song or artist currently playing by hovering over the album artwork, clicking the white up-arrow and selecting New Station—you can choose From Song or From Artist."
+         "fieldName" :  "scrollableMessageBody",
+     "fieldText" : "Create a Station.\nEnter an artist, song or composer in the Search box in the top left corner. We'll create a radio station featuring that music and more like it. You can also create a new station from the song or artist currently playing by hovering over the album artwork, clicking the white up-arrow and selecting New Station—you can choose From Song or From Artist."
```

Comment:
`"fieldName"` and `"fieldText"` should be aligned

Reply:
Resolved in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/155/commits/f84f198b7223d6940c1d2103ca3965f8d29ed8a4